universal-login-react/src/ui/WalletSelector/WalletSelector.tsx

```diff
@@ -77,16 +79,19 @@ export const WalletSelector = ({
     }
   };
 
-  const renderSuggestions = () =>
-    !busy && (connections.length || creations.length) ? (
+  const renderSuggestions = () => {
+    actions.includes(WalletSuggestionAction.connect) && ensureNotNull(onConnectClick, MissingParameter, 'onConnectClick');
+    actions.includes(WalletSuggestionAction.create) && ensureNotNull(onCreateClick, MissingParameter, 'onCreateClick');
```

Comment:
Optional suggestion:
Consider to throw when:
- actions do not include `WalletSuggestionAction.connect` but `onConnectClick` is provided
- actions do not include `WalletSuggestionAction.create` but `onCreateClick ` is provided